gems/pending/Gemfile

```diff
@@ -80,9 +80,9 @@ unless ENV['APPLIANCE']
   group :test do
     gem "camcorder",                    :require => false
     gem "coveralls",                    :require => false
-    gem "jasmine",       "~>2.4.0",     :require => false
-    gem "jasmine-core",  "~>2.4.0",     :require => false  # force the expected version, jasmine/jasmine-gem#271
-    gem "phantomjs",     "=1.9.8.0",    :require => false
+    gem "jasmine",                      :require => false
+    gem "jasmine-core",                 :require => false
```

Comment:
Can constrain if you want @jrafanie but for a test group (and non-appliance to boot) I didn't really care. What do you want, `~> 2.5`?